src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDescriptorProvider.cs

```diff
@@ -14,6 +14,8 @@ public class TagHelperDescriptorProvider
     {
         public const string CatchAllDescriptorTarget = "*";
```

Comment:
If we stick with two properties, should probably rename to `ElementCatchAllTarget` or something more clearly distinct from `RequiredAttributeWildcardSuffix`.